test/parallel/test-inspector-connect-main-thread.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+common.skipIfWorker();
```

Comment:
Why do we skip this if the process is a Worker thread? We should definitely also test what happens if this code is run inside a Worker, if we can. For now, this check breaks the test, because it doesn’t allow running this file in a Worker, not even from the test itself.

If there’s concern about the `isMainThread` check below, you can use a solution based on environment variables, e.g. the `process.env.HAS_STARTED_WORKER` bit in `test/parallel/test-worker-exit-code.js`. That’s a standard way of dealing with it in our test suite.

Reply:
This test will not work in a worker as it expects the code to be ran on the main thread. I updated the check.